bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -239,11 +238,25 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
     </div>
     <p><a href="#top">{{_('Back to Top')}}</a></p>
 </section>
+
+<!-- dislayed to nonFx users -->
+<div class="not-supported">
+  <header id="main-feature">
+      <h1 class="large">{{_('Check Your Plugins')}}</h1>
+      <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
+  </header>
+
+  <section>
+    <p>
```

Comment:
Ah yes, that makes more sense. I just followed the examples in the page blindly ;) difference of course is they have an else fallback. Nice catch.